.travis.yml

```diff
@@ -10,8 +10,6 @@ before_install:
 install:
     - ./install.sh miniconda
     - export PATH=$PWD/miniconda/bin:$PATH
-    - pip install -r requirements-pip.txt
-    - pip install -r tests/requirements.txt
     - if [[ $TRAVIS_PYTHON_VERSION == '2.6' ]]; then pip install unittest2 ordereddict counter; fi
```

Comment:
You can remove the python 2.6 specific stuff since conda is 2.7
I'd almost say you could remove the 3+ things as well and just put robotframework into requirements-pip.txt